integrations/sonarqube/client.py

```diff
@@ -468,6 +472,39 @@ async def get_all_sonarqube_analyses(
             )
             yield analysis_data
 
+    async def _get_all_portfolios(self) -> list[dict[str, Any]]:
+        logger.info(f"Fetching all portfolios in organization: {self.organization_id}")
+        response = await self.send_api_request(endpoint=Endpoints.PORTFOLIOS)
+        return response.get("views", [])
+
+    async def get_portfolio_details(self, portfolio_key: str) -> dict[str, Any]:
+        logger.info(f"Fetching portfolio details for: {portfolio_key}")
+        response = await self.send_api_request(
+            endpoint=Endpoints.PORTFOLIO_DETAILS,
+            query_params={"key": portfolio_key},
+        )
+        return response
+
+    def _get_portfolio_subportfolios(
+        self, portfolio: dict[str, Any]
+    ) -> list[dict[str, Any]]:
+        logger.info(f"Fetching subportfolios for: {portfolio['key']}")
+        subportfolios = portfolio.get("subViews", []) or []
```

Comment:
```suggestion
        subportfolios = portfolio.get("subViews", [])
```

Reply:
It is intentional. We encountered a bug recently on Kubecost integration where in the event a key has an explicit value of `None` it will return that instead of returning the default set in the `get` method call.

Reply:
But the whole purpose of getting the sub views is just so we can iteratively return it's details. So portfolio.get("subViews", []) will ensure that if the subViews is None, we just return an empty list. the additional [] seems redundant